### DISCUSSION TITLE
Is the "Sharp CAT" reference correct?

### DISCUSSION THREAD

**Discussion by blue-j** [colour-science/colour#1134] [Q&A] [answered]
The "Sharp" CAT cites:   [`https://www.researchgate.net/publication/227725635_Mathematical_approach_for_predicting_non-negative_tristimulus_values_using_the_CAT02_chromatic_adaptation_transform`](https://www.researchgate.net/publication/227725635_Mathematical_approach_for_predicting_non-negative_tristimulus_values_using_the_CAT02_chromatic_adaptation_transform)

Is this an error?  Why is it called "Sharp"?  It doesn't correspond to anything I can find.  Sorry if I am being daft!

[https://colour.readthedocs.io/en/latest/generated/colour.CHROMATIC_ADAPTATION_TRANSFORMS.html](https://colour.readthedocs.io/en/latest/generated/colour.CHROMATIC_ADAPTATION_TRANSFORMS.html])

[https://colour.readthedocs.io/en/v0.4.2/generated/colour.adaptation.CAT_SHARP.html](https://colour.readthedocs.io/en/v0.4.2/generated/colour.adaptation.CAT_SHARP.html)

J

**Comment by thomasmansencal** [comment, marked as answer]
Hi @blue-j,

Thanks for checking, the reference is correct:

![image](https://user-images.githubusercontent.com/99779/232252328-905173c7-4a01-42d7-80fb-77036a96c0bb.png)

Cheers,

Thomas
